src/libs/actions/BankAccounts.js

```diff
@@ -314,115 +315,131 @@ function goToWithdrawalAccountSetupStep(stepID, achData) {
+            failedValidationAttemptsName = CONST.NVP.FAILED_BANK_ACCOUNT_VALIDATIONS_PREFACE + bankAccountID;
+
+            // Now that we have the bank account. Lets grab the rest of the bank info we need
+            promiseAllSettled([
+                API.Get({
+                    returnValueList: 'nameValuePairs',
+                    name: failedValidationAttemptsName,
+                }),
+                API.Get({
+                    returnValueList: 'nameValuePairs',
+                    name: 'expensify_migration_2020_04_28_RunKycVerifications',
+                }),
+                API.Get({
+                    returnValueList: 'nameValuePairs',
+                    name: CONST.NVP.ACH_DATA_THROTTLED,
+                }),
+                API.Get({returnValueList: 'bankAccountList'}),
+            ])
+                .then(([
+                    failedValidationAttemptsResponse,
+                    kycVerificationsMigrationResponse,
+                    achDataThrottledResponse,
+                    bankAccountListResponse,
+                ]) => {
+                    const failedValidationAttempts = lodashGet(failedValidationAttemptsResponse, [
+                        'value', 'nameValuePairs', failedValidationAttemptsName,
+                    ], 0);
+                    const kycVerificationsMigration = lodashGet(kycVerificationsMigrationResponse, [
+                        'value', 'nameValuePairs', 'expensify_migration_2020_04_28_RunKycVerifications',
+                    ], '');
+                    const throttledDate = lodashGet(achDataThrottledResponse, [
+                        'value', 'nameValuePairs', CONST.NVP.ACH_DATA_THROTTLED,
+                    ], '');
+                    const bankAccountJSON = _.find(
+                        lodashGet(bankAccountListResponse, ['value', 'bankAccountList'], []), account => (
+                            account.bankAccountID === bankAccountID
+                        ),
+                    );
+                    const bankAccount = bankAccountJSON ? new BankAccount(bankAccountJSON) : null;
+
+                    // Next we'll build the achData and save it to Onyx
+                    // If the user is already setting up a bank account we will continue the flow for them
+                    let currentStep = previousACHData.currentStep;
+                    const achData = bankAccount ? bankAccount.toACHData() : {};
+                    achData.useOnfido = true;
+                    achData.policyID = '';
+                    achData.isInSetup = !bankAccount || bankAccount.isInSetup();
+                    achData.bankAccountInReview = bankAccount && bankAccount.isVerifying();
+                    achData.domainLimit = 0;
+                    achData.isDomainUsingExpensifyCard = false; // @TODO - Not actually sure if we need this
+
+                    // @TODO This subStep is used to either show the Plaid "login" view or the "manual" view - but not sure if
```

Comment:
many of these TODO were removed so we will need to merge `main` into this branch to update everything.